src/ca/wolfram/beta/symath/BaseNode.java

```diff
@@ -0,0 +1,24 @@
+package ca.wolfram.beta.symath;
```

Comment:
i feel like we can directly use the different subclass instead of relying on this one class